config/secrets_example.yml

```diff
@@ -3,8 +3,8 @@ development:
   APP_URL: http://localhost:9090
   TZ: Asia/Taipei
   # Security
-  DB_KEY: 
-  MSG_KEY: 
+  DB_KEY: <32-bit base64 string>
```

Comment:
Do we have a rake task to generate these keys?  If so, perhaps we can say something like `<use rake db:key>` as the value string.

Reply:
and same for `MSG_KEY`

Reply:
oh I see – it's in `doc/getting_start.md` - might be worth mentioning here too?  up to you!

Reply:
ok ~ I will change this part